### PR TITLE
Update using-testnet.md

### PR DESCRIPTION
**Pull request by RichardRed0x**
updating Decrediton testnet instructions

### CODE REVIEW

**Comment by jholdstock** [comment]
Hi Richard,

In all other places when we describe OS specific paths, we also include macOS. Could you add that please.

Could you also wrap the paths in backticks (`) rather than quotation marks (").

It might be nice actually to list the paths in a table, for easier readability. An example of this can be seen here:

https://docs.decred.org/getting-started/user-guides/decrediton-setup/

**Comment by RichardRed0x** [comment]
Copied that table over, I agree it's nicer.

File: docs/getting-started/using-testnet.md [hunk 1]
```diff
@@ -20,7 +20,13 @@ Running a testnet2 node is incredibly easy. You application of choice will need
 
 ## Decrediton 
 
-You can switch Decrediton between mainnet and testnet by changing the "Network" setting in the Settings tab, and then pressing the save button. You will have to close Decrediton and open it again for the change to take effect.
+You can switch Decrediton between mainnet and testnet by editing the config.json file and setting network: testnet. Config.json can be located in the following folders:
+
+| OS      | Decrediton data directory                           |
+| -------:|:---------------------------------------------------:|
+| Windows | `C:\Users\<your-username>\AppData\Local\Decrediton` |
```
Comment (hypernoob) [inline docs/getting-started/using-testnet.md]: I suggest changing this to ```| Windows | `%localappdata%\Decrediton` |```

**Comment by RichardRed0x** [comment]
Done

**Pull request merged**